packages/cypress/src/pageObjects/AppMessage/index.ts

```diff
@@ -0,0 +1,71 @@
+/// <reference types="cypress" />
+export interface IAppMessage {
+    shouldHaveMessage(message: string): Cypress.Chainable<JQuery<HTMLElement>>;
+    shouldBeVisible(): Cypress.Chainable<JQuery<HTMLElement>>;
+    shouldBeHidden(): Cypress.Chainable<JQuery<HTMLElement>>;
+    closeButton: Cypress.Chainable<JQuery<HTMLButtonElement>>;
+}
+
+/**
+ * The AppMessage page object allows to perform actions on the AppMessage component
+ * of `@rainbow-modules/app` library.
+ * @class
+ * @implements {IAppMessage}
+ * @type {AppMessage}
+ */
+class AppMessage implements IAppMessage {
+    private rootElement: string;
+
+    /**
+     * Constructs a new instance of this page object.
+     * @param rootElement The selector for the root element of the Button.
+     */
+    constructor(rootElement: string) {
+        this.rootElement = rootElement;
+    }
+
+    /**
+     * Asserts if the message passed is the current app message
+     * @method
+     * @param {string} error - The message text, that must match the current app message
+     * @returns {Cypress.Chainable<JQuery<HTMLElement>>}
+     */
+    shouldHaveMessage(message: string): Cypress.Chainable<JQuery<HTMLElement>> {
```

Comment:
we will change the way of doing assertions inside the page object and instead the page object will have an `expect` method
```
const app = new RainbowFirebaseApp();
app.message.expect('variant', 'error');
app.message.expect('message', 'some text');
app.message.expect('isVisible', true);
```